js/customer.js

```diff
@@ -0,0 +1,95 @@
+/*================= Form validation =================*/
+window.addEventListener('load', function () {
+    submitForm.addEventListener('click', function (event) {
+        event.preventDefault();
+        const name = document.getElementById('name');
+        const email = document.getElementById('email');
+        const phoneNumber = document.getElementById('phoneNumber');
+        function validEmail (email) {
+            return email.value.indexOf("@") !== 0 && email.value.indexOf("@") !== -1 && email.value.indexOf(".") !== -1 && email.value.lastIndexOf("@") < email.value.lastIndexOf(".") && email.value.lastIndexOf(".") < (email.value.length - 2) 
```

Comment:
can you google "Regular Expressions" to validate the email. Find a Regex that validates and use it. If you need explanation of how it works, then let us know.